scripts/azure-ci-test.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export GROUP_NAME="${GROUP_NAME:-ratify-e2e-$(openssl rand -hex 2)}"
+export ACR_NAME="${ACR_NAME:-ratifyacr$(openssl rand -hex 2)}"
+export AKS_NAME="${AKS_NAME:-ratify-aks-$(openssl rand -hex 2)}"
+export LOCATION="eastus"
+KUBERNETES_VERSION=${1:-1.24.6}
+GATEKEEPER_VERSION=${2:-3.11.0}
+TENANT_ID=$3
+RATIFY_NAMESPACE=${4:-default}
+CERT_DIR=${5:-"~/ratify/certs"}
+
+build_push_to_acr() {
+  echo "Building and pushing images to ACR"
+  docker build --progress=plain --no-cache -f ./httpserver/Dockerfile -t "${ACR_NAME}.azurecr.io/test/localbuild:test" .
+  docker push "${ACR_NAME}.azurecr.io/test/localbuild:test"
```

Comment:
If multiple PRs run at the same time that modify the httpserver, this will result in conflict. Add some randomness prefix to this tag too.

Reply:
The `ACR_NAME` is generated randomly, shouldn't that avoid the conflict?

Reply:
Added randomized tag as well.